src/test/java/com/bobocode/svydovets/bibernate/session/SessionTest.java

```diff
@@ -143,6 +141,7 @@ void mergeDetachedEntityWhenEntityInCache() {
         mockedEntityStateService = Mockito.mock(EntityStateServiceImpl.class);
         setInternalDependency(session, "entityStateService", mockedEntityStateService);
 
+        // todo: fix it. Returns null entity. Why magic 123L?
         session.find(Person.class, 123L); // Put entity into cache
```

Comment:
That was the default id from factory, which was changed to 1
Nice To use DEFAULT_ID like in other methods
session.find(Person.class, DEFAULT_ID); Mb this will fix that